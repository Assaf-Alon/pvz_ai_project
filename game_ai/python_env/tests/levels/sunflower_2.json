{
    "10": [["Normal", 3]],
    "20": [["Normal", 3]],
    "30": [["Normal", 3]],
    "60": [["Normal", 3]]
}
