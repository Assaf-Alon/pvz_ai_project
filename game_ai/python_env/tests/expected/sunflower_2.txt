[0] Planted Sunflower in (4, 4). Total: 0.
[50] Sun generated by Sunflower in (4, 4). Total: 25.
[100] Sun generated by Sunflower in (4, 4). Total: 50.
[100] Normal zombie being spawned at lane 3
[100] Sun autospawned. Total: 75.
[101] Planted Sunflower in (1, 1). Total: 25.
[147] Normal zombie in (3, 9) moved to (3, 8).
[150] Sun generated by Sunflower in (4, 4). Total: 50.
[151] Sun generated by Sunflower in (1, 1). Total: 75.
[176] Planted Sunflower in (1, 0). Total: 25.
[194] Normal zombie in (3, 8) moved to (3, 7).
[200] Sun generated by Sunflower in (4, 4). Total: 50.
[200] Normal zombie being spawned at lane 3
[200] Sun autospawned. Total: 75.
[201] Sun generated by Sunflower in (1, 1). Total: 100.
[226] Sun generated by Sunflower in (1, 0). Total: 125.
[241] Normal zombie in (3, 7) moved to (3, 6).
[247] Normal zombie in (3, 9) moved to (3, 8).
[250] Sun generated by Sunflower in (4, 4). Total: 150.
[251] Sun generated by Sunflower in (1, 1). Total: 175.
[251] Planted Sunflower in (2, 1). Total: 125.
[276] Sun generated by Sunflower in (1, 0). Total: 150.
[288] Normal zombie in (3, 6) moved to (3, 5).
[294] Normal zombie in (3, 8) moved to (3, 7).
[300] Sun generated by Sunflower in (4, 4). Total: 175.
[300] Normal zombie being spawned at lane 3
[300] Sun autospawned. Total: 200.
[301] Sun generated by Sunflower in (1, 1). Total: 225.
[301] Sun generated by Sunflower in (2, 1). Total: 250.
[326] Sun generated by Sunflower in (1, 0). Total: 275.
[335] Normal zombie in (3, 5) moved to (3, 4).
[341] Normal zombie in (3, 7) moved to (3, 6).
[347] Normal zombie in (3, 9) moved to (3, 8).
[350] Sun generated by Sunflower in (4, 4). Total: 300.
[351] Sun generated by Sunflower in (1, 1). Total: 325.
[351] Sun generated by Sunflower in (2, 1). Total: 350.
[376] Sun generated by Sunflower in (1, 0). Total: 375.
[382] Normal zombie in (3, 4) moved to (3, 3).
[388] Normal zombie in (3, 6) moved to (3, 5).
[394] Normal zombie in (3, 8) moved to (3, 7).
[400] Sun generated by Sunflower in (4, 4). Total: 400.
[400] Sun autospawned. Total: 425.
[401] Sun generated by Sunflower in (1, 1). Total: 450.
[401] Sun generated by Sunflower in (2, 1). Total: 475.
[426] Sun generated by Sunflower in (1, 0). Total: 500.
[429] Normal zombie in (3, 3) moved to (3, 2).
[435] Normal zombie in (3, 5) moved to (3, 4).
[441] Normal zombie in (3, 7) moved to (3, 6).
[450] Sun generated by Sunflower in (4, 4). Total: 525.
[451] Sun generated by Sunflower in (1, 1). Total: 550.
[451] Sun generated by Sunflower in (2, 1). Total: 575.
[476] Sun generated by Sunflower in (1, 0). Total: 600.
[476] Normal zombie in (3, 2) moved to (3, 1).
[482] Normal zombie in (3, 4) moved to (3, 3).
[488] Normal zombie in (3, 6) moved to (3, 5).
[500] Sun generated by Sunflower in (4, 4). Total: 625.
[500] Sun autospawned. Total: 650.
[501] Sun generated by Sunflower in (1, 1). Total: 675.
[501] Sun generated by Sunflower in (2, 1). Total: 700.
[523] Normal zombie in (3, 1) moved to (3, 0).
[526] Sun generated by Sunflower in (1, 0). Total: 725.
[529] Normal zombie in (3, 3) moved to (3, 2).
[535] Normal zombie in (3, 5) moved to (3, 4).
[550] Sun generated by Sunflower in (4, 4). Total: 750.
[551] Sun generated by Sunflower in (1, 1). Total: 775.
[551] Sun generated by Sunflower in (2, 1). Total: 800.
[570] Normal zombie in (3, 0) attempted entering the house.
[570] Normal zombie in (3, 0) triggered a lawnmower.
[570] Lawnmower in (3, 0) attacked.
[570] Normal zombie in (3, 0) was damaged. HP: -9818.
[570] Normal zombie in (3, 0) was killed.
[571] Lawnmower in (3, 0) moved to (3, 1).
[573] Lawnmower in (3, 1) moved to (3, 2).
[573] Lawnmower in (3, 2) attacked.
[573] Normal zombie in (3, 2) was damaged. HP: -9818.
[573] Normal zombie in (3, 2) was killed.
[575] Lawnmower in (3, 2) moved to (3, 3).
[576] Sun generated by Sunflower in (1, 0). Total: 825.
[577] Lawnmower in (3, 3) moved to (3, 4).
[577] Lawnmower in (3, 4) attacked.
[577] Normal zombie in (3, 4) was damaged. HP: -9818.
[577] Normal zombie in (3, 4) was killed.
[579] Lawnmower in (3, 4) moved to (3, 5).
[581] Lawnmower in (3, 5) moved to (3, 6).
[583] Lawnmower in (3, 6) moved to (3, 7).
[585] Lawnmower in (3, 7) moved to (3, 8).
[587] Lawnmower in (3, 8) moved to (3, 9).
[589] Lawnmower in (3, 9) moved to (3, 10).
[589] Lawnmower in (3, 10) removed.
[600] Sun generated by Sunflower in (4, 4). Total: 850.
[600] Normal zombie being spawned at lane 3
[600] Sun autospawned. Total: 875.
[601] Sun generated by Sunflower in (1, 1). Total: 900.
[601] Sun generated by Sunflower in (2, 1). Total: 925.
[626] Sun generated by Sunflower in (1, 0). Total: 950.
[647] Normal zombie in (3, 9) moved to (3, 8).
[650] Sun generated by Sunflower in (4, 4). Total: 975.
[651] Sun generated by Sunflower in (1, 1). Total: 1000.
[651] Sun generated by Sunflower in (2, 1). Total: 1025.
[676] Sun generated by Sunflower in (1, 0). Total: 1050.
[694] Normal zombie in (3, 8) moved to (3, 7).
[700] Sun generated by Sunflower in (4, 4). Total: 1075.
[700] Sun autospawned. Total: 1100.
[701] Sun generated by Sunflower in (1, 1). Total: 1125.
[701] Sun generated by Sunflower in (2, 1). Total: 1150.
[726] Sun generated by Sunflower in (1, 0). Total: 1175.
[741] Normal zombie in (3, 7) moved to (3, 6).
[750] Sun generated by Sunflower in (4, 4). Total: 1200.
[751] Sun generated by Sunflower in (1, 1). Total: 1225.
[751] Sun generated by Sunflower in (2, 1). Total: 1250.
[776] Sun generated by Sunflower in (1, 0). Total: 1275.
[788] Normal zombie in (3, 6) moved to (3, 5).
[800] Sun generated by Sunflower in (4, 4). Total: 1300.
[800] Sun autospawned. Total: 1325.
[801] Sun generated by Sunflower in (1, 1). Total: 1350.
[801] Sun generated by Sunflower in (2, 1). Total: 1375.
[826] Sun generated by Sunflower in (1, 0). Total: 1400.
[835] Normal zombie in (3, 5) moved to (3, 4).
[850] Sun generated by Sunflower in (4, 4). Total: 1425.
[851] Sun generated by Sunflower in (1, 1). Total: 1450.
[851] Sun generated by Sunflower in (2, 1). Total: 1475.
[876] Sun generated by Sunflower in (1, 0). Total: 1500.
[882] Normal zombie in (3, 4) moved to (3, 3).
[900] Sun generated by Sunflower in (4, 4). Total: 1525.
[900] Sun autospawned. Total: 1550.
[901] Sun generated by Sunflower in (1, 1). Total: 1575.
[901] Sun generated by Sunflower in (2, 1). Total: 1600.
[926] Sun generated by Sunflower in (1, 0). Total: 1625.
[929] Normal zombie in (3, 3) moved to (3, 2).
[950] Sun generated by Sunflower in (4, 4). Total: 1650.
[951] Sun generated by Sunflower in (1, 1). Total: 1675.
[951] Sun generated by Sunflower in (2, 1). Total: 1700.
[976] Sun generated by Sunflower in (1, 0). Total: 1725.
[976] Normal zombie in (3, 2) moved to (3, 1).
[1000] Sun generated by Sunflower in (4, 4). Total: 1750.
[1000] Sun autospawned. Total: 1775.
[1001] Sun generated by Sunflower in (1, 1). Total: 1800.
[1001] Sun generated by Sunflower in (2, 1). Total: 1825.
[1023] Normal zombie in (3, 1) moved to (3, 0).
[1026] Sun generated by Sunflower in (1, 0). Total: 1850.
[1050] Sun generated by Sunflower in (4, 4). Total: 1875.
[1051] Sun generated by Sunflower in (1, 1). Total: 1900.
[1051] Sun generated by Sunflower in (2, 1). Total: 1925.
[1070] Normal zombie in (3, 0) attempted entering the house.
[1070] You've lost! Zombie entered at lane 3
